File: veterinarskastanicamobile/lib/model/usluge.dart
import 'package:json_annotation/json_annotation.dart';

part 'usluge.g.dart';

@JsonSerializable()
class Service {
  int? uslugaId;
  String? naziv;
  String? sifra;
  double? cijena;
  int? vrstaId;
  int? jedinicaMjereId;
  String? slika;
  bool? status;

  Service();

  factory Service.fromJson(Map<String, dynamic> json) =>
      _$ServiceFromJson(json);

  Map<String, dynamic> toJson() => _$ServiceToJson(this);
}

File: VeterinarskaStanica.Model/Usluge.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace VeterinarskaStanica.Model
{
	public partial class Usluge
	{
		public int UslugaId { get; set; }

		public string Naziv { get; set; }

		public string Sifra { get; set; }

		public decimal Cijena { get; set; }

		public string Opis { get; set; }
		public int VrstaId { get; set; }

		public int JedinicaMjereId { get; set; }


		public byte[] Slika { get; set; }

		public byte[] SlikaThumb { get; set; }

		public bool? Status { get; set; }
		public string PaymentId { get; set; }

		public string StateMachine { get; set; }

	}
}
